Wireframe/index.html

```diff
@@ -8,26 +9,45 @@
   </head>
   <body>
     <header>
-      <h1>Wireframe</h1>
+      <h1>Wireframe to web code</h1>
       <p>
-        This is the default, provided code and no changes have been made yet.
+        Welcome to my ITP Onboarding Module Wirefare to web code.
       </p>
     </header>
     <main>
       <article>
         <img src="placeholder.svg" alt="" />
```

Comment:
Can you review the documentation for the `<img>` element? It has 2 required attributes.
https://www.w3schools.com/tags/tag_img.asp